pyspider/fetcher/tornado_fetcher.py

```diff
@@ -64,7 +64,7 @@ def size(self):
 
 
 class Fetcher(object):
-    user_agent = "pyspider/%s (+http://pyspider.org/)" % pyspider.__version__
+    user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/28.0.1500.52 Safari/537.36"
```

Comment:
please don't change the default UA, you can do that in project scripts, but for pyspider, it's not responsible to use a fake UA.